packages/lesswrong/lib/collections/typingIndicators/schema.ts

```diff
@@ -0,0 +1,28 @@
+import { userOwns } from '../../vulcan-users/permissions';
+
+const schema: SchemaType<DbTypingIndicator> = {
+  userId: {
+    type: String,
+    optional: true,
+    nullable: false,
+    canRead: ['members'],
+    canCreate: ['members'],
+  },
+  documentId: {
+    type: String,
+    optional: true,
```

Comment:
Remove `optional: true` for these; that's a field that's only relevant for generating forms and we aren't generating forms based on TypingIndicators